global.json

```diff
@@ -1,5 +1,5 @@
 {
-  "msbuild-sdks": {
-    "MSBuild.Sdk.Extras": "1.6.55"
+  "sdk": {
+    "version": "3.0.100-preview-010046"

```

Comment:
There should be a script to install this (like the required SDKs)

Reply:
The build server does it with the yaml file, I'll investigate how to do it from a PS script.

Reply:
I added a ps1 for that.